lib/provider/data_types.go

```diff
@@ -123,3 +123,9 @@ type VolumeAuthorization struct {
 	// List of HostIPs to authorize
 	HostIPs []string `json:"hostIPs,omitempty"`
 }
+
+// VolumeList ...
+type VolumeList struct {
+	Next    string    `json:"next,omitempty"`
```

Comment:
Is this Next required?, how this will be consumed by caller

Reply:
It is required by CSI driver's `ListVolumes` call which in subsequent calls, will start listing volumes from the volID subpplied by `Next` field.
https://github.com/container-storage-interface/spec/blob/master/lib/go/csi/csi.pb.go#L2281